### ISSUE TITLE
Queries for streets with hyphens in the name

### ISSUE THREAD

**Issue by orangejulius** [pelias/pelias#738]
When querying for a street or address on a street that has a hyphen in the name, it's often not possible to find the desired result if that hyphen is omitted. This is common for highways and related roads in the United States.

Some examples:

## 3590 I-80 Service Rd, Laramie County, WY, USA

With hyphen, no issues:
[3590 I-80 Service Rd, Laramie County, WY, USA](http://pelias.github.io/compare/#/v1/search%3Ftext=3590%20I-80%20Service%20Rd,%20Laramie%20County,%20WY,%20USA)
![screenshot_2018-08-16_19-43-28](https://user-images.githubusercontent.com/111716/44240533-aa552400-a18c-11e8-9680-0d400f01d0e1.png)

Without the hyphen (I80 instead of I-80). Strangely, in this case the street is found,but not the address from OpenAddresses
[/v1/search?text=3590 I80 Service Rd, Laramie County, WY, USA](http://pelias.github.io/compare/#/v1/search%3Ftext=3590%20I80%20Service%20Rd,%20Laramie%20County,%20WY,%20USA)
![screenshot_2018-08-16_19-46-53](https://user-images.githubusercontent.com/111716/44240634-264f6c00-a18d-11e8-80e7-c5fb68113898.png)

## West I-65 Service Road South, Mobile, AL

[/v1/search/structured?address=W I-65 service rd s&locality=Mobile&region=Alabama](http://pelias.github.io/compare/#/v1/search/structured%3Faddress=W%20I-65%20service%20rd%20s&locality=Mobile&region=Alabama)
![screenshot_2018-08-16_19-51-45](https://user-images.githubusercontent.com/111716/44240760-d4f3ac80-a18d-11e8-9076-3d0ed3d68b44.png)

Without the hyphen the query falls back to the locality
[/v1/search/structured?address=W I65 service rd s&locality=Mobile&region=Alabama](http://pelias.github.io/compare/#/v1/search/structured%3Faddress=W%20I65%20service%20rd%20s&locality=Mobile&region=Alabama)
![screenshot_2018-08-16_19-52-34](https://user-images.githubusercontent.com/111716/44240774-efc62100-a18d-11e8-9bef-550921cd6960.png)

**Comment by missinglink** [comment]
Would this be resolved by https://github.com/pelias/schema/pull/310 ?

**Comment by orangejulius** [comment]
I don't think so. Looking at the integration tests for the street analyzer, [hyphens are still included](https://github.com/pelias/schema/pull/310/files#diff-4411781cb48cb864e1345ab568a32181R18).

We _could_ look at removing hyphens. Or even better, like mentioned in https://github.com/pelias/schema/issues/65#issuecomment-237240507, we could use our now fully functional alt-name support to handle both the case with a hyphen and where the hyphen is replaced with a space.
